fixtures/shehds/led-flat-par-12x3w-rgbw.json

```diff
@@ -55,20 +55,13 @@
       }
     },
     "Strobe": {
-      "capabilities": [
+      "capability":
         {
-          "dmxRange": [0, 0],
-          "type": "ShutterStrobe",
-          "shutterEffect": "Open"
-        },
-        {
-          "dmxRange": [1, 255],
           "type": "ShutterStrobe",
           "shutterEffect": "Strobe",
           "speedStart": "0Hz",
           "speedEnd": "10Hz"
         }
-      ]
```

Comment:
But then how is strobe disabled? At which DMX values is the light constantly on or off?